### ISSUE TITLE
mysql:5.6 is missing mysql_config_editor

### ISSUE THREAD

**Issue by brownoxford** [docker-library/mysql#105]
The mysql_config_editor program is missing from the mysql:5.6 and mysql:5.7 containers. This is a client program, and should be included wherever the CLI is included.

**Comment by yosifkit** [comment]
Unfortunately it is not included in any of the packages in the apt repo provided by MySQL,  but it looks like it will soon be included in the packaging: https://bugs.mysql.com/bug.php?id=73670#c436020

**Comment by tianon** [comment]
It's not included in 5.6, but it _is_ included in 5.7, so I'm inclined to consider this issue fixed:
```console
$ docker run -it --rm mysql:5.6 mysql_config_editor
/usr/local/bin/docker-entrypoint.sh: line 134: exec: mysql_config_editor: not found
$ docker run -it --rm mysql:5.7 mysql_config_editor
mysql_config_editor Ver 1.0 Distrib 5.7.14, for Linux on x86_64
Copyright (c) 2012, 2016, Oracle and/or its affiliates. All rights reserved.

Oracle is a registered trademark of Oracle Corporation and/or its
affiliates. Other names may be trademarks of their respective
owners.

MySQL Configuration Utility.
Usage: mysql_config_editor [program options] [command [command options]]
  -#, --debug[=#]     This is a non-debug version. Catch this and exit.
  -?, --help          Display this help and exit.
  -v, --verbose       Write more information.
  -V, --version       Output version information and exit.

Variables (--variable-name=value)
and boolean options {FALSE|TRUE}  Value (after reading options)
--------------------------------- ----------------------------------------
verbose                           FALSE

Where command can be any one of the following :
       set [command options]     Sets user name/password/host name/socket/port
                                 for a given login path (section).
       remove [command options]  Remove a login path from the login file.
       print [command options]   Print all the options for a specified
                                 login path.
       reset [command options]   Deletes the contents of the login file.
       help                      Display this usage/help information.

```

**Comment by ltangvald** [comment]
As noted in the upstream bug, in the 5.6 packaging mysql_config_editor is in libmysqlclient-dev instead of the client (it was fixed in 5.7.14, but not yet in 5.6)
